processing/stenosis_connector_aux_test.go

```diff
@@ -30,11 +30,11 @@ type mockGRPCServerOptionFunc func(*mockGRPCServer)
 
 type mockGRPCServerTokenGenFunc func(*task.Query) string
 
-func mockGRPCServerFailOption(meth string, err error) mockGRPCServerOptionFunc {
-	return func(m *mockGRPCServer) {
-		m.failWith[meth] = err
-	}
-}
+//func mockGRPCServerFailOption(meth string, err error) mockGRPCServerOptionFunc {
```

Comment:
Always good to comment code that is being commented out. Otherwise it looks like forgotten debugging code, or dead code.

Reply:
It is indeed dead code, kept around because I do not know Remo's intention for it. Removed it as it can be reproduced from Git history if necessary.